docs/source/reference/series.rst

```diff
@@ -70,3 +70,10 @@ Missing data handling
    Series.isna
    Series.notna
    Series.dropna
+
+Serialization / IO / Conversion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. autosummary::
+   :toctree: api/
+
+   Series.to_numpy
```

Comment:
Seems like pandas' doc has `to_num` in "Conversion" section above in this file.
Shall we move this up there?

Reply:
I think it's better we keep it in `Serialization / IO / Conversion` because the `Conversion` section contains methods used to clean data like `isna()` or make a copy `copy()`. Somehow they are not relevant to `to_numpy()`.

In `Conversion` section, we have methods used to convert the DataFrame/Series like `DataFrame.to_parquet()`, `DataFrame.to_json()`

@ueshin  what do you think ?

Reply:
I'd prefer to follow pandas' way, but not a strong opinion. cc @thunterdb

Reply:
I think that either points are valid and I do not have a strong opinion because the name of the method is so explicit and because it is very easy to change later. Since the pull request is structured as it is currently, I would be of the opinion that we merge it as it is. Also, nothing prevents us from adding the reference in both places. @ueshin I will let you decide.

Reply:
then I'd merge this now. thanks for the opinion!